### PR TITLE
Added winget-gui

### PR DESCRIPTION
**Pull request by danielehrhardt**
This is the link to the project: [Winget GUI](https://github.com/codextde/winget-gui)

- [x] **I have read the [contributing guidelines](contributing.md).**
- [x] Use the following format: `[Title](link) - Description.`
- [x] Add entries to the bottom of the correct category.
- [x] No unnecessary words like `for Tauri`, `a Tauri plugin` and `Super-Fast`.
- [x] Description does not start with `A` or `An`.
- [x] Description is under 24 words.
- [x] Description has no links or parenthesis.
- [x] Use `backticks` when mentioning package names.
- [x] Double-check spelling and grammar.
- [x] No trailing whitespace is added to the end of any file.
- [x] One suggestion per pull request.
- [x] I understand that my entry will be removed if it violates the guidelines.
- [x] I have [signed my commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### Plugins/Integrations

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The project is open source and accepts contributions.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The project is active and maintained.

### Templates

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The template provides enough information about how to get started and what's included.
- [ ] The template is pretty different from the existing templates.

### Apps

- [x] The app is original and not too simple.
- [x] The README is in English.
- [x] The app makes a reasonable effort to be fast, lightweight and secure.
- [x] If the app closed source, evidence of it being built with Tauri is included.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [GitBar](https://github.com/mikaelkristiansson/gitbar) - System tray app for GitHub reviews.
 - [Wirefish](https://github.com/stefanodevenuto/wirefish) - Cross-platform packet sniffer and analyzer.
 - [Kanri](https://github.com/trobonox/kanri) - Cross-platform, offline-first Kanban board app with a focus on simplicity and user experience.
+- [Winget GUI](https://github.com/codextde/winget-gui) - Open-source winget gui
```
Comment (JonasKruckenberg) [inline README.md]:
```suggestion
- [Winget GUI](https://github.com/codextde/winget-gui) - Open-source winget GUI.
```

**Review by JonasKruckenberg — CHANGES_REQUESTED** [review]
You may also consider adding a CSP here https://github.com/codextde/winget-gui/blob/319e5b29da9d401265ea2f90e2c1eb00a3c6be3a/src-tauri/tauri.conf.json#L57
And also get rid of the demo greet command 😄 https://github.com/codextde/winget-gui/blob/319e5b29da9d401265ea2f90e2c1eb00a3c6be3a/src-tauri/src/main.rs#L6-L10

**Comment by danielehrhardt** [comment]
code changes done

**Review by JonasKruckenberg — APPROVED** [review]

**Comment by JonasKruckenberg** [comment]
Oh actually, we require all commits to be signed across Tauri's repositories. Follow [this guide](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) to set up commit signing. Once all the commits in this PR are signed we can merge your contribution. Thanks for being part of our community!

**Comment by simonhyll** [comment]
@danielehrhardt We have updated the structure of the README a bit. Please update your commit to reflect this change. Also, as already mentioned, all commits need to be signed before we can accept them.

**Pull request closed without merging**